Makefile

```diff
@@ -1,81 +1,8 @@
 NAME    := isa-l
+DEB_NAME := libisal
 SRC_EXT := gz
 SOURCE   = https://github.com/01org/$(NAME)/archive/v$(VERSION).tar.$(SRC_EXT)
-PATCHES := $(NAME)-553f01f.patch
+#PATCHES := $(NAME)-553f01f.patch
```

Comment:
Why are we commenting this out?

Reply:
I commented it out during testing as it is a reverse patch.

When I went to prepare a forward patch, I discovered that DAOS was now specifying the release with out any patches.

Reply:
Ah, nice!.  So let's just delete the line, and the file.  #2 should have done that, but since it didn't this is a good opportunity to clean that up.